push.sh

```diff
@@ -11,7 +11,11 @@ then
   echo
   echo "Pushing release commits and tag"
   echo "------------------------------------------------------------------------------"
-  git push && git push --tags
+  if ! (git push && git push --tags); then
```

Comment:
Can you change it to push specific refs. 
```
git push origin HEAD:refs/heads/${branch}
git push origin <tag>
```

(not sure what variable we have for tag